axon-server/administration/admin-configuration/configuration.md

```diff
@@ -0,0 +1,1072 @@
+      <td style="text-align:left">replication.min-active-backups</td>
+      <td style="text-align:left">When active backup nodes are added to a context, this indicates on how
+        many active backup nodes transactions must be confirmed before the transaction
+        is ready for commit.</td>
+      <td style="text-align:left">1</td>
+    </tr>
+    <tr>
+      <td style="text-align:left"></td>
+      <td style="text-align:left">replication.min-election-timeout</td>
+      <td style="text-align:left">Minimal time (in ms.) that a follower waits before moving to candidate
+        state, if it has not received any messages from a leader.</td>
+      <td style="text-align:left">1000</td>
+    </tr>
+    <tr>
+      <td style="text-align:left"></td>
+      <td style="text-align:left">replication.primary-cleanup-delay</td>
+      <td style="text-align:left">Windows only. Delay before forcing the primary segment file from memory
+        when no longer in use.</td>
+      <td style="text-align:left">5</td>
+    </tr>
+    <tr>
+      <td style="text-align:left"></td>
+      <td style="text-align:left">replication.secondary-cleanup-delay</td>
+      <td style="text-align:left">Windows only. Delay before forcing the other segment files from memory
+        when no longer in use.</td>
+      <td style="text-align:left">30</td>
+    </tr>
+    <tr>
+      <td style="text-align:left"></td>
+      <td style="text-align:left">replication.segment-size</td>
+      <td style="text-align:left">Size of a transaction log file.</td>
+      <td style="text-align:left">16MB</td>
+    </tr>
+    <tr>
+      <td style="text-align:left"></td>
+      <td style="text-align:left">replication.snapshot-flow-buffer</td>
+      <td style="text-align:left">Number of unconfirmed install snapshot messages that may be sent to peer.</td>
+      <td
+      style="text-align:left">50</td>
+    </tr>
+    <tr>
+      <td style="text-align:left"></td>
+      <td style="text-align:left">replication.sync-interval</td>
+      <td style="text-align:left">Interval to check for files that can be closed.</td>
+      <td style="text-align:left">1000</td>
+    </tr>
+    <tr>
+      <td style="text-align:left"></td>
+      <td style="text-align:left">replication.wait-for-leader-timeout</td>
+      <td style="text-align:left">Timeout to wait for leader when requesting access to event store while
+        leader change in progress, if not set defaults to maxElectionTimeout.</td>
+      <td
+      style="text-align:left">-1</td>
+    </tr>
+    <tr>
+      <td style="text-align:left"><em><b>Keep Alive</b></em>
+      </td>
+      <td style="text-align:left"></td>
+      <td style="text-align:left"></td>
+      <td style="text-align:left"></td>
+    </tr>
+    <tr>
+      <td style="text-align:left"></td>
+      <td style="text-align:left">keep-alive-time</td>
+      <td style="text-align:left">Interval at which AxonServer will send timeout messages. Set to 0 to disbable
```

Comment:
Would be nice to state the time unit here, which I believe to be ms.
And the following:
```suggestion
      <td style="text-align:left">Interval at which AxonServer will send timeout messages. Set to 0 to disable
```